app/assets/stylesheets/shared_styles/elements/buttons.scss

```diff
@@ -0,0 +1,155 @@
+// scss-lint:disable NestingDepth
+
+.sci-btn {
+  @include font-button;
+  animation-timing-function: $timing-function-sharp;
+  background: $brand-primary;
+  border: 1px solid $brand-primary;
+  border-radius: $border-radius-default;
+  color: $color-white;
+  cursor: pointer;
+  display: inline-block;
+  font-size: 14px;
+  line-height: 20px;
+  outline: 0;
+  padding: 7px 16px;
+  position: relative;
+  text-decoration: none;
+  transition: .3s;
+  user-select: none;
+
+  .fas {
+    color: inherit;
+    margin-right: 5px;
+  }
+
+  &:hover {
+    background: $brand-primary-hover;
+    color: $color-white;
+    text-decoration: none;
+  }
+
+  &:active {
+    background: $brand-primary-press;
+    color: $color-white;
+    text-decoration: none;
+  }
+
+  &:focus {
+    border: $border-focus;
+    color: $color-white;
+    outline: 0;
+    text-decoration: none;
+  }
+
+  &.secondary {
```

Comment:
It would be really nice, if we would add the naming of elements to the style-guide document.

Reply:
Yes